browser/ui/views/tabs/features.h

```diff
@@ -20,6 +29,14 @@ extern const base::Feature kBraveVerticalTabs;
 // Returns true when users chose to use vertical tabs
 bool ShouldShowVerticalTabs();
 
+// Returns true when we should show title bar on window frame when vertical
+// tab strip is enabled.
+bool ShouldShowTitleBarOnVerticalTabs(const Browser* browser);
```

Comment:
Rename this to ShouldShowWindowTitleOnVerticalTabs()

WindowTitleBar and Window title could mean different in the context of views implementation. WindowTitleBar contains Caption button and icon.